spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubOperations.java

```diff
@@ -97,4 +99,34 @@
 	 */
 	Subscriber subscribe(String subscription, MessageReceiver messageHandler);
 
+	/**
+	 * Pulls a number of messages from a Google Cloud Pub/Sub subscription.
+	 *
+	 * <p>
+	 * Automatically acknowledges received messages.
+	 *
+	 * @param pullRequest request object containing request settings
+	 * @return the list of received messages
+	 */
+	List<PubsubMessage> pull(PullRequest pullRequest);
```

Comment:
Hmmm why does the method signature have to symmetric?

I wanted to avoid exposing a new kind of response like `PullResponse`. Users only knew `PubsubMessage` before.